tpl/template_funcs.go

```diff
@@ -1699,13 +1699,21 @@ func countRunes(content interface{}) (int, error) {
 }
 
 // humanize returns the humanized form of a single word.
+// If the word given can be identified as an integer, the behavior
+// is to add the corresponding ordinal.
 // Example:  "my-first-post" -> "My first post"
+// Example:  "103" -> "103rd"
 func humanize(in interface{}) (string, error) {
 	word, err := cast.ToStringE(in)
 	if err != nil {
 		return "", err
 	}
-	return inflect.Humanize(word), nil
+
+	_, err = strconv.Atoi(word)
```

Comment:
Hrm... I had structured this way for two reasons. I was trying to support strings that were integer-y i.e. `"103"` but hadn't considered the case of a literal `int` being passed. But also because `inflect.Ordinalize` expects a string param, so we'll need to do that conversion regardless of if we end up calling `inflect.Ordinalize` or `inflect.Humanize`.
It's simple enough to add the logic for handling literal `int` arguments to the template func. I'll add that.